WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserActivity.java

```diff
@@ -352,8 +352,9 @@ private void activateTheme(String themeId) {
             @Override
             public void onResponse(JSONObject response) {
                 WordPress.wpDB.setCurrentTheme(siteId, newThemeId);
+                Theme newTheme = WordPress.wpDB.getTheme(siteId, newThemeId);
+                AnalyticsUtils.trackWithCurrentBlogDetails(AnalyticsTracker.Stat.THEMES_CHANGED_THEME);
```

Comment:
Can we add the `id` of the theme the user has switched to as property of the event? 
Can we make sure the `blog_id` is added to properties as well?